### PR TITLE
fix: colorpicker move on input

### LINKED ISSUE #654: ColorPicker Area Moves Unexpectedly with Mouse Scroll and Keyboard Inputs
**Issue Summary**:
The color picker area within the ColorPicker moves unexpectedly upon gaining focus and receiving certain inputs.

### Expected Behavior

Upon gaining focus, the color picker area within the ColorPicker should remain static, regardless of mouse or keyboard inputs.

### Current Behavior

Once the color picker area gains focus:
1. Scrolling using the mouse's middle button causes the area to move.
2. Pressing keys such as Home, End, PageUp, PageDown, Left, Right, Up, and Down also causes the color picker area to move.

### Steps to Reproduce

1. Open the ColorPicker component.
2. Focus on the color picker area.
3. Use the mouse's middle button to scroll or press any of the specified keyboard keys.

### Your Environment

- SceneBuilder version: (8),20,21
- Operating System: Windows 10
- Java version: (8),20, 21

### Screenshots

**GIF**
move_area
**PNG**
image

### PR DESCRIPTION
**Pull request by leewyatt**
**PR Summary**:
This PR addresses the unexpected movement of the ColorPicker area when it gains focus and certain mouse or keyboard inputs are made.

### Issue

Fixes #654 

### Progress

- [x] Change must not contain extraneous whitespace
- [x] License header year is updated, if required
- [x] The PR name must follow the [pre-defined format](https://github.com/gluonhq/scenebuilder/blob/master/CONTRIBUTING.md)
- [x] Verify the contributor has signed [Gluon Individual Contributor License Agreement (CLA)](https://docs.google.com/forms/d/16aoFTmzs8lZTfiyrEm8YgMqMYaGQl0J8wA0VJE2LCCY)

### CODE REVIEW

**Review by jperedadnr — COMMENTED** [review]
Looks good, and fixes the issue.
Please just move the two event filters right before the `// paint combobox...` comment, including a new empty line or a new comment after the `liveUpdateListener` lines.

**Comment by leewyatt** [comment]
@jperedadnr Thank you for reviewing my pull request and for your valuable feedback. I've updated the code according to your suggestions. Please let me know if there are any further adjustments needed.

File: kit/src/main/java/com/oracle/javafx/scenebuilder/kit/util/control/paintpicker/colorpicker/ColorPicker.java [hunk 1]
```diff
@@ -252,6 +256,11 @@ private void initialize() {
         picker_region.pressedProperty().addListener(liveUpdateListener);
         hue_slider.pressedProperty().addListener(liveUpdateListener);
         alpha_slider.pressedProperty().addListener(liveUpdateListener);
+
+        // Fix ColorPicker movement on focus. Now blocks scroll and key events (Home, End, PgUp, PgDn, arrows) to keep it static. For issue #654.
```
Comment (jperedadnr) [inline kit/src/main/java/com/oracle/javafx/scenebuilder/kit/util/control/paintpicker/colorpicker/ColorPicker.java:260]: Looks good, just a small rewording:

```suggestion
        // Disable ColorPicker movement on focus, by blocking scroll and key (Home, End, PgUp, PgDn, arrows) events
```

**Comment by leewyatt** [comment]
Thank you for the suggestion! I've committed the changes as per your review.

**Review by jperedadnr — DISMISSED** [review]
looks good!

**Comment by jperedadnr** [comment]
Oh, one more little change is required before we can merge this PR, we need to update the copyright year of the modified file.

Please change:

```
 * Copyright (c) 2022, Gluon and/or its affiliates.
```
into:
```
 * Copyright (c) 2022, 2024, Gluon and/or its affiliates.
```

**Review by jperedadnr — APPROVED** [review]

**Pull request merged**